class.jetpack.php

```diff
@@ -3010,7 +3022,7 @@ public static function deactivate_module( $module ) {
 
 		$jetpack = Jetpack::init();
 
-		$active = Jetpack::get_active_modules();
+		$active = Jetpack_Options::get_option( 'active_modules', array() );
```

Comment:
I'm not sure why we need this change - could you elaborate? Also, should we consider fixing this in `get_active_modules()` instead?

Reply:
Good catch, since update_active_modules() already takes care of only updating incremental values, we can leave this unchanged. Implemented in [23a9976](https://github.com/Automattic/jetpack/pull/12270/commits/23a9976e3220ec4ebae74eebe9881e4016aaf5d1).